lib/detect-testing-library-utils.ts

```diff
@@ -96,6 +96,8 @@ export interface DetectionHelpers {
   isQuery: IsQueryFn;
   isCustomQuery: IsCustomQueryFn;
   isAsyncUtil: IsAsyncUtilFn;
+  isFireEventUtil: (node: TSESTree.Identifier) => boolean;
```

Comment:
I defined types inline here instead of extracting them. I'm not sure what was the advantage of that (not having to write it twice?) but both ways are pretty awful. I may refactor all types here to inline types.